### PR TITLE
Add AvailabilitySubscriber component

### PR DESCRIPTION
**Pull request by andreldsa**
#### What is the purpose of this pull request?
Integrate the `AvailabilitySubscriber` component

#### How should this be manually tested?
Acces: https://andre--storecomponents.myvtex.com/porsche-911/p

#### Screenshots or example usage
image

#### Types of changes

* [ ] Bug fix (a non-breaking change which fixes an issue)
* [x] New feature (a non-breaking change which adds functionality)
* [ ] Breaking change (fix or feature that would cause existing functionality to change)
* [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

**Review by salesfelipe — APPROVED** [review]

File: react/ProductDetails.js [hunk 1]
```diff
@@ -28,6 +29,15 @@ class ProductDetails extends Component {
   handleSkuChange = skuIndex => {
     this.setState({ skuIndex })
   }
+  
+  compareSku = (item1, item2) => {
```
Comment (andreldsa) [inline react/ProductDetails.js]: Moved from the button of the file.

Comment (estacioneto) [reply react/ProductDetails.js]: How about something like...

```javascript
const [
  {
    commertialOffer: {
      AvailableQuantity: AvailableQuantity1,
      Price: Price1,
    },
  },
] = item1.sellers
const [
  {
    commertialOffer: {
      AvailableQuantity: AvailableQuantity2,
      Price: Price2,
    },
  },
] = item2.sellers
```

**Review by estacioneto — APPROVED** [review]

**Review by lucasecdb — APPROVED** [review]

File: react/ProductDetails.js [hunk 2]
```diff
@@ -49,10 +49,18 @@ class ProductDetails extends Component {
       )
     }
 
+    const initialItem = product.items[0].itemId
```
Comment (estacioneto) [inline react/ProductDetails.js]:
```javascript
const [ { itemId: initialItem } ] = product.items
```

File: react/ProductDetails.js (hunk 1 above)
Comment (andreldsa) [reply react/ProductDetails.js]: Top

**Pull request merged**